contracts/swaps/connectors/testnet/SwapsImplLocal.sol

```diff
@@ -10,6 +10,7 @@ import "../../../openzeppelin/SafeERC20.sol";
 import "../../ISwapsImpl.sol";
 import "../../../feeds/IPriceFeeds.sol";
 import "../../../testhelpers/TestToken.sol";
+import "hardhat/console.sol";
```

Comment:
Once testing is over, we should remove the console logs from the smart contract.

Reply:
of course I will